app/controllers/items_controller.rb

```diff
@@ -229,10 +229,16 @@ def edit_copyright
     @change_set = ItemChangeSet.new(@cocina)
   end
 
+  # Draw form for barcode
+  def edit_barcode
+    @change_set = ItemChangeSet.new(@cocina)
+  end
+
   # save the copyright form
   def update
     change_set = ItemChangeSet.new(@cocina)
     change_set.validate(copyright: params[:item][:copyright])
+    change_set.validate(barcode: params[:item][:barcode])
```

Comment:
There is no need for a second call to validate, just pass in all the arguments at once

Reply:
Just saw this in your license PR, I'll update and/or rebase depending on which goes in first.